### DISCUSSION TITLE
What is the differnence between "best_model" and "best_model_$step"?

### DISCUSSION THREAD

**Discussion by ruaruaruabick** [coqui-ai/TTS#1708] [General Q&A]
While training a TTS model, I find two best models in my output dir. What is the differnence between "best_model" and "best_model_$step"?

**Comment by mbarnig** [comment]
`best_model.pth` file is a copy of the latest `best_model_$step.pth` file.

**Reply by thorstenMueller** [reply]
In addition to what @mbarnig said, you can configure your training model with a parameter call `keep_all_best` / `save_all_best` to keep all best models for complete training. If you set it to `false` training will overwrite best_model file and you've just access to the latest one.
